supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc

```diff
@@ -75,6 +75,17 @@ Use the hyphenated "fault-tolerant" when using it as an adjective.
 
 *See also*:
 
+//Add FedRAMP terminology Issue #395
+[[fedramp]]
+==== image:images/yes.png[yes] FedRAMP® (noun)
+*Description*: The Federal Risk and Authorization Management Program (FedRAMP®) is managed by the FedRAMP Program Management Office. Use the ® symbol at first usage in a document. For additional information, see the link:https://www.fedramp.gov/assets/resources/documents/FedRAMP_Branding_Guidance.pdf[FedRAMP Brand Guide].
```

Comment:
@spank50 A handy tip is to put each sentence on a new line, then it is very obvious which sentence a reviewer is referring to when they add a comment. :) 

```suggestion
*Description*: The Federal Risk and Authorization Management Program (FedRAMP) is managed by the FedRAMP Program Management Office. For additional information, see the link:https://www.fedramp.gov/assets/resources/documents/FedRAMP_Branding_Guidance.pdf[FedRAMP Brand Guide].
```
Check with Red Hat Legal re. whether we must use the registered trademark symbol (®) with FedRAMP in RH docs, or is it covered by the generic trademark statement?

Reply:
The FedRAMP brand guide requires (R) symbol at first significant use in text.
I will confirm with Legal team that the US Gov't requirement for (R) is valid.

Reply:
@spank50 Any update on this?

Reply:
Sentence line breaks added per suggestion from bredamc.
We'll follow the Federal guidelines for (R) usage at first major use in text (not required in title or section heads).
I think we're good to go.